derp/derp_server.go

```diff
@@ -190,6 +199,9 @@ type Server struct {
 	// maps from netip.AddrPort to a client's public key
 	keyOfAddr map[netip.AddrPort]key.NodePublic
 
+	// Sets the client send queue depth
```

Comment:
before-line comments are typically full sentence (https://go.dev/wiki/CodeReviewComments#comment-sentences) ... see lines 187 and 193 above. Ending with periods too.